tests/AccountServiceClientTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+use Accounts\V1\DeactivationType;
+use Vendasta\Accounts\V1\AccountsServiceClient;
+use Vendasta\Accounts\V1\DeactivateAppRequest;
+use Vendasta\Vax\SDKException;
+
+class AccountServiceClientTest extends TestCase
+{
+    public function testDeactivateApp()
+    {
+        $environment = getenv("ENVIRONMENT");
+        if ($environment == null) {
+            $environment = "DEMO";
+        }
+        $client = new AccountsServiceClient($environment);
+
+        $req = new DeactivateAppRequest();
+        $req->setBusinessId("AG-P3KH8VTHR5");
+        $req->setAppId("MP-MCLBF8TMSMLCQ5W72LNHQV5XVW4KC2MX");
```

Comment:
No one else running this test will have it pass with these values. Could maybe leave them open ended like `MP-EXAMPLE`, just to signify that